### PR TITLE
Feature/agent finishes

### PR DESCRIPTION
**Pull request by abist**
Full Bug List -
https://github.com/Microsoft/sqlopsstudio/issues/1162

Fixes
- [x] Fixed a bug in the Jobs View page where if you clicked on the left most part of the row, it would crash or show a black screen.
- [x] Added icons for steps. 
- [x] Fix styling of Jobs button in Page selection panel.
- [x]  Add horizonal rule line after Jobs label to match mock-up.
- [x] Remove "Category ID"
- [x] Switch "true/false" to "yes/no" or a simple checkbox icon or something pending UX (optional)
- [x] Jobs View status indicator color should match status of last run outcome.
- [x]  No vertical scrollbar on Jobs View grid.
- [x] Jobs View "Next Run" column (and really any date column) should not show uninitialized DateTime. Match SSMS behavior.
- [x] Steps list hover background isn't correct for Light Theme.
- [x] All error messages, particularly the Job Execution details Message, should be copyable to clipboard. Either as selectable text or hover action or otherwise.
- [x] Reduce line spacing for Job Execution pane execution details label.
- [x] Up\Down arrows in Job History list should update Job Execution details (i.e., no Enter required).
- [x]  Using keyboard with Job History list, enter on element doesn't update Job Execution details.
- [x]  Pressing TAB when focus Job History Execution list causes weird styling

### CODE REVIEW

File: src/sql/parts/jobManagement/views/jobStepsView.component.ts [hunk 1]
```diff
@@ -17,14 +17,15 @@ import { DashboardServiceInterface } from 'sql/parts/dashboard/services/dashboar
 import { AgentJobHistoryInfo } from 'sqlops';
 import { JobStepsViewController, JobStepsViewDataSource, JobStepsViewFilter,
 	JobStepsViewRenderer, JobStepsViewRow, JobStepsViewModel} from 'sql/parts/jobManagement/views/jobStepsViewTree';
+import { JobHistoryComponent } from './jobHistory.component';
```
Comment (kburtram) [inline src/sql/parts/jobManagement/views/jobStepsView.component.ts]: do we have any coding conventions for using relative vs. full import paths?  I typically use the full path, which seems like the case for all the other imports in this file.

File: src/sql/parts/jobManagement/views/jobsView.component.ts [hunk 2]
```diff
@@ -129,20 +129,8 @@ export class JobsViewComponent implements AfterContentChecked {
...
-					'itemDetail': job,
-				}, undefined, null);
-			},
-			panelRows: 2,
-			postTemplate: () => ''
-		});
-
-		this.rowDetail = rowDetail;
 
+		this.rowDetail = new RowDetailView({});
```
Comment (kburtram) [inline src/sql/parts/jobManagement/views/jobsView.component.ts]: why don't we need the RowDetailView configuration parameters?  How are we showing the error messages on the Jobs List page?

Comment (abist) [reply src/sql/parts/jobManagement/views/jobsView.component.ts]: I'm gonna send another PR for the implementation of errors. This is to get rid of the bug for now.

File: src/sql/parts/jobManagement/views/jobStepsViewTree.ts [hunk 3]
```diff
@@ -46,6 +46,7 @@ export class JobStepsViewController extends TreeDefaults.DefaultController {
 	private _jobHistories: AgentJobHistoryInfo[];
 
 	protected onLeftClick(tree: tree.ITree, element: JobStepsViewRow, event: IMouseEvent, origin: string = 'mouse'): boolean {
+		//tree.expand(element);
```
Comment (kburtram) [inline src/sql/parts/jobManagement/views/jobStepsViewTree.ts]: dead code?

File: src/sql/parts/jobManagement/views/jobsView.component.ts [hunk 4]
```diff
@@ -49,16 +50,15 @@ export class JobsViewComponent implements AfterContentChecked {
...
-		{ name: 'Last Run', field: 'lastRun' },
-		{ name: 'Next Run', field: 'nextRun' },
-		{ name: 'Enabled', field: 'enabled' },
-		{ name: 'Status', field: 'currentExecutionStatus' },
-		{ name: 'Category', field: 'category' },
-		{ name: 'Runnable', field: 'runnable' },
-		{ name: 'Schedule', field: 'hasSchedule' },
-		{ name: 'Category ID', field: 'categoryId' },
-		{ name: 'Last Run Outcome', field: 'lastRunOutcome' },
+		{ name: 'Name', field: 'name', formatter: this.renderName, width: 200 },
```
Comment (kburtram) [inline src/sql/parts/jobManagement/views/jobsView.component.ts]: should the column headers be localized strings?

**Pull request merged**